README.md

```diff
@@ -0,0 +1,51 @@
+## TwinCAT Documentation Template
+
+This project is intended to be used from PCDS's CI scripts, but it may be used
+standalone as well.
+
+### How this works
+
+The make target ``html`` has been reconfigured to generate documentation
+first with ``ads-deploy``.
+
+``ads-deploy`` will automatically be installed in the current Python
+environment if it is unavailable, along with other requirements found in
+[requirements.txt](/requirements.txt) in this repository.
+
+```bash
+$ make html
+# Installs requirements.txt if ads-deploy not found
+# Generates source/*.rst with solutions from TWINCAT_PROJECT_ROOT
+# Builds build/html
+```
+
+By default, TWINCAT_PROJECT_ROOT is `..`. This means that the documentation
+template is expected to be cloned *inside* the top level of the PLC project.
+
+### As a submodule
+
+This could be used as a submodule in repositories:
```

Comment:
Have we done this anywhere? What would the benefits be?

Reply:
We only use this repository as part of CI and refer to the master branch for documentation generation and have not utilized this approach.

This would more closely reflect how we do things in Python land. The upside to including it in the project is that it's explicit as to how documentation generation relates to the source code (i.e., it isn't linked just by a CI job that you have to jump through hoops to figure out). The downside is that the repository maintainer has to keep track of the documentation generation process, which is an additional (arguably unnecessary) burden.

Reply:
I remember encountering submodules and straying away from them due to the extra effort required to keep them up-to-date.  I still think this is a nice inclusion, though